openmetadata-ui/src/main/resources/ui/src/utils/ServiceUtilClassBase.ts

```diff
@@ -140,7 +142,11 @@ class ServiceUtilClassBase {
     };
   }
 
-  public getServiceTypeLogo(type: string) {
+  public getServiceTypeLogo(
+    type: string,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _searchSource?: SearchSuggestions[number] | SearchSourceAlias
```

Comment:
Why we want this unused variable here?

Reply:
@chirag-madlani, some entities will not have the serviceType so we need to have searchSource to get the service logo for the entity, it is unused here but will be used in the extended class.